extensions/amp-bind/amp-bind.md

```diff
@@ -55,9 +59,11 @@ limitations under the License.
   </tr>
 </table>
 
-## Overview
+## Usage
+
+The `amp-bind` component enables custom stateful interactivity on AMP pages.
 
-The `amp-bind` component allows you to add custom stateful interactivity to your AMP pages via data binding and JS-like expressions.
+For performance and to avoid the risk of unexpected content jumping, `amp-bind` does not evaluate expressions on page load. This means that the visual elements should be given a default state and not rely `amp-bind` for initial render.
```

Comment:
suggestion: turn this into an `Important:` call out

Reply:
I have a theory that user's skim over the callouts. I think things that are this important should just be explained. I don't think reference docs should have a ton of callouts....

https://www.nngroup.com/articles/banner-blindness-old-and-new-findings/